### ISSUE TITLE
Questions for raw data to data

### ISSUE THREAD

**Issue by unknowed-ER** [huangtinglin/Knowledge_Graph_based_Intent_Network#22]
Thank you for adding the knowledge map to the original data, but I wonder if all the sequences of the test set and training set are sorted by timestamp? 
Can you provide code to process raw data or a timestamp for each interaction?

**Comment by sunnysq** [comment]
I also want to know if all the sequences of the test set and training set are sorted by timestamp?

**Comment by huangtinglin** [comment]
Sorry for the late reply. Actually, most of the dataset used in our paper is organized in the other project [KGAT](https://github.com/xiangwang1223/knowledge_graph_attention_network). I have no idea the details of these datasets since I haven't participated in that project.
